scripts/deploy-helm.sh

```diff
@@ -389,11 +389,24 @@ echo "- 7. Install dynatrace-gcp-function with helm chart in $CLUSTER_NAME"
 helm install ./dynatrace-gcp-function --generate-name --namespace dynatrace --set clusterName="$CLUSTER_NAME" > ${CMD_OUT_PIPE}
 
 echo
-echo "- Deployment complete, check if containers are running:"  > ${CMD_OUT_PIPE}
+echo -e "\e[92m- Deployment complete, check if containers are running:\e[37m"  > ${CMD_OUT_PIPE}
 if [[ $DEPLOYMENT_TYPE == logs ]] || [[ $DEPLOYMENT_TYPE == all ]]; then
   echo "kubectl -n dynatrace logs -l app=dynatrace-gcp-function -c dynatrace-gcp-function-logs"  > ${CMD_OUT_PIPE}
 fi
 
 if [[ $DEPLOYMENT_TYPE == metrics ]] || [[ $DEPLOYMENT_TYPE == all ]]; then
   echo "kubectl -n dynatrace logs -l app=dynatrace-gcp-function -c dynatrace-gcp-function-metrics"  > ${CMD_OUT_PIPE}
+fi
+
+if [[ $DEPLOYMENT_TYPE == all ]]; then
+  # To build a Log viewer link we need a Dynatrace url which is set only for 'metrics' and 'all' deployment types.
+  # For 'logs' deployment type (where ActiveGate url is used) we are not able to build the link - LOG_VIEWER is empty then.
```

Comment:
By default, we are using the containerized AG, thus dynatraceURL is known since we are using it to pull AG image from the given cluster. So to summarize the Log_viewer URL **can't** be constructed only for the deployment type 'logs'  with the use of existing AG enabled.